packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -20,7 +20,7 @@
         class="oc-topbar-unauthenticated-avatar oc-flex-inline oc-flex-center oc-flex-middle"
         :name="$gettext('User Menu login')"
         :width="32"
-        icon="user-add"
+        icon="user"
```

Comment:
now user can login, go to preferences so its not only create account (which also doesnt exist :D)